.gitignore

```diff
@@ -7,3 +7,5 @@ env
 .poll-change-timestamp.json
 fetched*
 src/helmet.js
+src/config.js
```

Comment:
`src/config.js` should be included in version control (Configuration variables are from environment variables where all the sensitive data are). See [Helmet harvester](https://github.com/NatLibFi/melinda-record-import-harvester-helmet/blob/dev/src/config.js) for refence.